playground/mocks/config/responseConfig.js

```diff
@@ -825,7 +825,7 @@ const secondaryEmail = {
   '/idp/idx/identify': [
     'enroll-profile-update-params',
     'enroll-profile-update-all-optional-params',
-    'error-enroll-profile-update-params'
+    'enroll-profile-update-params'
```

Comment:
just fixing the name? or should a mock be added/removed?

Reply:
@pradeepdewda-okta - Given most of these form fields are dynamic, we should probably defer all translations to the API layer.

IMO, it would be a bit strange to only support translations for `secondaryEmail`, but none of the others.